app/controllers/initiatives_controller.rb

```diff
@@ -4,7 +4,7 @@ class InitiativesController < ApplicationController
   before_action :find_initiative, only: [:edit, :update, :destroy]
 
   def index
-    @initiatives = Initiative.all
+    @initiatives = Initiative.all.order("completion ASC")
```

Comment:
The `order` method works in ascending order by default, so it's not necessary to specify here. You might also consider changing completion to a symbol, ie `Initiative.all.order(:completion)`. There is a good reason for this and I'm struggling to remember what it is... 😬Generally speaking though, when I use Active Record I try to avoid SQL strings unless it's totally necessary for a complicated query, because Active Record provides useful errors and I think will error if it is passed a symbol that it doesn't recognize. @thatonejason, can you add to this? My Rails is rusty!

Reply:
Completion is changed to a symbol